src/status_im/contexts/wallet/wallet_connect/utils/data_transformations.cljs

```diff
@@ -0,0 +1,73 @@
+(ns status-im.contexts.wallet.wallet-connect.utils.data-transformations
```

Comment:
in other places we named this kind of file `data-store`, maybe we can use the same name here?

Reply:
thanks! i knew there was a "convention" in our project for these kind of files but couldn't find and example. Will rename it, cause it's also shorter.

Reply:
done :+1: